data/_ui/interfaces.txt

```diff
@@ -185,10 +185,10 @@ interface "map detail panel"
 interface "map planet card"
 	value "text start" 13
 	value "category size" 20
-	value "height" 130
+	value "height" 150
```

Comment:
just gets me thinking, since we have category size we should probably just have this be named "extra height" and in this case be 30? (150 - 6 * 20, since category size is already something we have)
we can also just name it margin

you dont have to do this though I could do it in another PR or submit a PR to this one

Reply:
I prefer "extra height"

Reply:
Do you need me to do it?
By the way just curious was this code easy enough to modify? It is a bit complex but I tried to make it easily modifiable

Reply:
It wasn't hard to show an additional row on all cards, but now we need to support variable card height, which requires a MapPlanetCard / MapDetailPanel rework, to make things like available space calculations don't assume fixed height.
If you could do that it would be really helpful.

Reply:
I think this needs a bit more thought put into it for how it'd be best to code it
The easiest way is probably to just figure out if we have different govs in the system and if we do display the gov of all planets? idk